src/util/Constants.js

```diff
@@ -205,6 +205,7 @@ const Endpoints = exports.Endpoints = {
     return {
       Emoji: emojiID => `${root}/emojis/${emojiID}.png`,
       Asset: name => `${root}/assets/${name}`,
+      DefaultAvatar: number => `${root}/embed/avatars/${number}.png`,
```

Comment:
The new url is nicer, yes, but the system breaks down if they ever choose to add any in the future, and you have to look in the user file/every other file it is used in and not the constants file to fix it if they ever do. But again, my original questioning didn't have to do with that aspect. 

It purely had to do with, ~"did they change the urls, because you are resolving something completely different" + "it didn't change" = RedFlag